docs/src/main/sphinx/release/release-406.md

```diff
@@ -66,6 +70,7 @@
 
 ## Iceberg connector
 
+* Add support for changing column types. ({issue}`15515`)
```

Comment:
Is there such a thing as a column type in Iceberg or in general .. or is it the data type of a column .. or is that just a longer name for the same thing

Reply:
Sorry, I don't understand the question. The entry means supporting `ALTER TABLE [ IF EXISTS ] name ALTER COLUMN column_name SET DATA TYPE new_type` statement in Iceberg connector.

Reply:
Sounds like is just the data type of a column .. all good. We ca leave as it.